src/tpm2tss_engine.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 Xaptum, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <openssl/ui.h>
+
+#ifndef ENFTUN_SSL_UI_H
+#define ENFTUN_SSL_UI_H
+UI_METHOD *
+enftun_ssl_ui_passthrough(void);
```

Comment:
Does `enftun_ssl_ui_passthrough` need to be in the public API.  It can probably just be a static function in `tpm.c`.

Reply:
This can be static.